package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcds-components",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Since the site-menu hasn't been fully updated yet and is in the main branch, lets hold off on updating the package until that is done.

Reply:
Done, changed it back to 0.1.2